pkgs/development/interpreters/php/update.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -I nixpkgs=../../../../ -i bash -p wget jq
+
+src=$(wget -qO- "https://www.php.net/releases/active" | jq '[ .[] | to_entries | map( { (.key): {version: .value.version, hash: .value.source[0].sha256, filename: .value.source[0].filename} } ) | add] | add')
```

Comment:
Why not just pipe it directly to the file at this point?

Reply:
It's not finished yet, I need to convert hashes to sri hashes. I will definitely do that as soon as I found how to execute a shell command in jq.

Reply:
At that point just write in in Python or something.